apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts

```diff
@@ -106,14 +110,12 @@ onDocumentReady(() => {
     },
     onPreBody: () => {
       $('#grading-table [data-toggle="popover"]').popover('dispose');
-      $('#grading-table [data-toggle="tooltip"]').tooltip('dispose');
     },
     onPostBody: () => {
       updateGradingTagButton();
       $('#grading-table [data-toggle="popover"]')
         .popover({ sanitize: false })
         .on('shown.bs.popover', updatePointsPopoverHandlers);
-      $('#grading-table [data-toggle=tooltip]').tooltip({ html: true });
```

Comment:
AFAICT the `html: true` was unnecessary. There's a single tooltip (open instance warning) and it doesn't use HTML in the tooltip content.